### PR TITLE
configurable parent mailer

### PR DESCRIPTION
**Pull request by mespina**
Parent Mailer for customizations

### CODE REVIEW

File: lib/templates/contact_us.rb [hunk 1]
```diff
@@ -27,4 +27,8 @@
   # Configure the redirect URL after a successful submission
   config.success_redirect = '/'
 
-end
+  # Configure the parent action mailer
+  # Example:
+  # config.parent_mailer = "ActionMailer::Base"
+
```
Comment (mespina) [inline lib/templates/contact_us.rb]: Fixed

File: lib/templates/contact_us.rb [hunk 2]
```diff
@@ -27,4 +27,8 @@
...
+end
```
Comment (mespina) [inline lib/templates/contact_us.rb]: Fixed

**Comment by JDutil** [comment]
Could you please delete and gitignore the .DS_Store file.

**Comment by JDutil** [comment]
Thanks

**Comment by JDutil** [comment]
Released v1.1.0 with your change.

**Pull request closed without merging**